### PR TITLE
Add ArchLinux dependency installation

### PR DESCRIPTION
**Pull request by PRESFIL**
Tested recently. Works without pip-packages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +30,17 @@ sudo modprobe v4l2loopback
...
+#### On ArchLinux
+All dependencies currenly packaged in base ArchLinux repositories.
+
+```
+sudo pacman -S python3 opencv python-numpy python-pillow hdf5 v4l2loopback-dkms
+sudo depmod -a
+sudo modprobe v4l2loopback
+```
+
+---
```
Comment (natesales) [inline README.md:42]: Lets remove the extra whitespace here

**Comment by natesales** [comment]
Thanks! The only thing before merging is there's some extra whitespace in the readme as reviewed above.

**Comment by PRESFIL** [comment]
Like that?

**Comment by natesales** [comment]
Looks good, merged.

**Pull request merged**